kura/distrib/src/main/resources/aarch64-nn/jdk.dio.properties

```diff
@@ -0,0 +1,31 @@
+#
+# Copyright (c) 2011, 2014, 2017 Eurotech and/or its affiliates
```

Comment:
CONTRIBUTING.md suggests the following date range format: year1, year2.

In this case it would be
```
# Copyright (c) 2011, 2017 Eurotech and/or its affiliates and others
...
```

Or, since it's actually a new file, this is more appropriate:

```
# Copyright (c) 2017 Eurotech and/or its affiliates and others
...
```

Reply:
I can let it as you like, as most of the files were just copied from other directories I prefered to left the original copyrights.